### ISSUE TITLE
on data changed event

### ISSUE THREAD

**Issue by urvashi-k-7span** [xsahil03x/multi_trigger_autocomplete#16]
I am using this with my custom user list model class and on tap I am returning that class because want to store selected user ids, but when I replace one user how can I replace that user id?
Can this have any particular events for replacement of mentioned user?

**Comment by xsahil03x** [comment]
Hey @urvashik-7span, I am not able to understand your requirement. Can you explain it a bit more? Or maybe some images or flow might help.

**Comment by urvashi-k-7span** [comment]
@xsahil03x 
I have used this package and MentionAutocompleteOptions I have called my user list from rest API that will have separate model class for receiving data and onMentionUserTap I have returned that class.

OnTap of that I have added user id into one list like

```
  List<int>? taggedUsersId = [];
 final autocomplete = MultiTriggerAutocomplete.of(context);
                            if ((taggedUsersId != null || (taggedUsersId?.isNotEmpty ?? false)) &&
                                (!taggedUsersId!.contains(user.id))) {
                              taggedUsersId?.add(user.id ?? 0);
                            }
```

Now when I mentioned one user one user id is added.
Then I wrote some text after that I mentioned another user another user id is added,

But when I tried to remove the previously/first mentioned user and select another user in my list, 3rd user id will be added.

How can I prevent this, I mean want to remove replaced mentioned user id.

How can I maintain a user id list with current text of text field?
